source/connection-monitoring-and-pooling/tests/pool-checkout-custom-maxConnecting-is-enforced.json

```diff
@@ -0,0 +1,81 @@
+{
+  "version": 1,
+  "style": "integration",
+  "description": "custom maxConnecting is enforced",
```

Comment:
This test LGTM, passes in python and fails if maxConnecting is not configurable.

Reply:
the test passed in c#